### PR TITLE
Project To-dos, Darcy Winterhill

### PR DESCRIPTION
**Pull request by ghost**
Link to my deployed project:
https://vacay-planner.netlify.app/

My reflections on how this week's project turned out:
I think it turned out OK and I'm happy with the result and the look of it.

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import moment from 'moment'
+import styled from 'styled-components'
+
+const Header = () => {
+  const date = moment().format('MMM Do');
```
Comment (Pauan8) [inline code/src/components/Header.js:7]: Though moment is very nice and useful, we might want to change this in the future as it's not maintained anymore :)

File: README.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
-# Project Todos
+Project done during the Technigo Bootcamp
 
-Replace this readme with your own information about your project.
+# Vacay Planner Project
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project was to do a todo web app using Redux, where you can add, delete and set tasks as done and not done.
+I turned it into a vacation planner.
```
Comment (Pauan8) [inline README.md:6]: Such a nice idea with a vacation planner! 👍

File: code/src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          {date}
+        </Info>
+        <Info>
+          {undoneTasks} to-do's
+        </Info> 
+      </InfoContainer>
+    </HeaderContainer>
+  )
+}
+
```
Comment (Pauan8) [inline code/src/components/Header.js:30]: Nice and clean code with the styled components, very descriptive names 💯

File: code/src/components/Input.js [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+  const [value, setValue] = useState('')
+
+  const dispatch = useDispatch()
+
+  const onFormSubmit = (e) => {
+    e.preventDefault()
+
+    const newToDo = {
+      id: uuidv4(),
```
Comment (Pauan8) [inline code/src/components/Input.js:18]: uuid, top notch 🥇

File: code/src/components/Input.js [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+          <DoneUndoneButton
+            type='button'
+            onClick={() => dispatch(todos.actions.allDone())}>
+              All done
+          </DoneUndoneButton>
+          <DoneUndoneButton
+            type='button'
+            onClick={() => dispatch(todos.actions.allUndone())}>
+              Reset
+          </DoneUndoneButton>
```
Comment (Pauan8) [inline code/src/components/Input.js:40]: These two could have easily been a function instead with parameter, as they are very similar.

File: code/src/components/Input.js [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+border-radius: 20%;
+width: 75px;
+height: 55px;
+box-shadow: 1px 2px 2px #646666;
+margin: 0 10px;
+padding: 0;
+&:hover {
+  cursor: pointer;
+}
+`
```
Comment (Pauan8) [inline code/src/components/Input.js]: These two buttons have many similar styles, an idea would be to use the same button for both and use props to set the specific values where they differ!

**Review by Pauan8 — APPROVED** [review]
Impressive project, very easy to follow. Clean and nice code and good use of lottie and styled components! 🥇 Keep up the great work!

File: README.md (hunk 2 above)
Comment (KarolCislowski) [reply README.md:6]: YES 😻 I haven't got a vacation for over 5 years! Maybe next year 😸

File: code/src/components/Header.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+const HeaderContainer = styled.div`
+display: flex;
+flex-direction: column;
+align-items: center;
+
+width: 80%;
+border-top-left-radius: 20px;
+border-top-right-radius: 20px;
+background-color: #1B2021;
+margin: auto;
```
Comment (KarolCislowski) [inline code/src/components/Header.js]: Where is the indentation?? 🙀

File: code/src/components/Input.js (hunk 6 above)
Comment (KarolCislowski) [reply code/src/components/Input.js]: True

**Review by KarolCislowski — APPROVED** [review]
Great styled app with nice functionalities 🌈

**Pull request closed without merging**